contrib/sip_proxy/filters/network/source/conn_manager.cc

```diff
@@ -281,7 +281,7 @@ void ConnectionManager::sendLocalReply(MessageMetadata& metadata, const DirectRe
   default:
     PANIC("not reached");
   }
-  stats_.counterFromElements("", "local-generated-response").inc();
+  // stats_.counterFromElements("", "local-generated-response").inc();
```

Comment:
I don't think removing metrics is a good idea. How is anyone going to operate this for real? In general also please do not leave commented out code.

Reply:
Hi @mattklein123, the CPU will grow 2% with metrics, we remove the metric temporarily for performance perspective. 
And I have removed the commented out code.  Please help to review the change, thanks!